core/handler/tournament-handler.go

```diff
@@ -8,15 +8,25 @@ import (
 	"github.com/kimbellG/kerror/kegrpc"
 	ttgrpc "github.com/kimbellG/tournament/core/handler/grpc"
 	"github.com/kimbellG/tournament/core/models"
+	log "github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 func (sh *ServiceHandler) CreateTournament(ctx context.Context, r *ttgrpc.CreateTournamentRequest) (*ttgrpc.CreateTournamentResponse, error) {
+	createLog := log.WithFields(log.Fields{
+		"action":  "create tournament",
+		"request": r,
+	})
+	createLog.Info("received create tournament request")
```

Comment:
log.WithFields(log.Fields{
	"action":  "create tournament",
	"request": 
}).Info("received create tournament request")

Reply:
i create logrus.Entry and then put this to ErrorLog function, also this require for eliminate duplicates when i'm logging result of request